elements/pf-dropdown/pf-dropdown.ts

```diff
@@ -65,11 +65,6 @@ export class PfDropdown extends LitElement {
    */
   @property({ type: Boolean, reflect: true }) disabled = false;
 
-  /**
-   * Flag to indicate if menu is opened.
-   */
-  @property({ type: Boolean, reflect: true }) expanded = false;
```

Comment:
Might still be useful to reflect this, e.g. for styling or to programmatically open the drop-down by setting the attribute